src/settings.ts

```diff
@@ -22,6 +22,9 @@ export const SPARKLE_TERMS_AND_CONDITIONS_URL =
 export const SPARKLE_PRIVACY_POLICY =
   "https://sparklespaces.com/privacy-policy/";
 
+// Sparkle facebook app id. More settings can be found at https://developers.facebook.com/apps/2633721400264126/dashboard/
+export const FACEBOOK_SPARKLE_APP_ID = "2633721400264126";
```

Comment:
@mike-lvov This absolutely shouldn't have been hardcoded and exposed in the source here like this.. That's the whole point/reason of the ENV vars mapped in through from `src/secrets.js`

Reply:
Clicking through to that link I have no access to the project. Can you please share that with me and the rest of the team who needs it.